.github/workflows/docker.yml

```diff
@@ -2,13 +2,18 @@ name: Docker Build and Push
 # Build & Push builds the simapp docker image on every push to master
 # and pushes the image to https://hub.docker.com/u/provenanceio
 on:
+  pull_request:
+    paths:
+      - docker/blockchain/Dockerfile
+      - .github/workflows/docker.yml
   push:
     branches:
       - main
     tags:
-      - "v[0-9]+.[0-9]+.[0-9]+" # Push events to matching v*, i.e. v1.0, v20.15.10
-      - "v[0-9]+.[0-9]+.[0-9]+-rc*" # Push events to matching v*, i.e. v1.0-rc1, v20.15.10-rc5
-      
```

Comment:
I'm pretty sure we need at least one entry here. We'd want `- "**"` to run on any tag, Alternatively, `- "*"` to run on tags that don't have a `/` in them.

I know you can leave the `pull_request` section empty to make a workflow run on any pull request. That's a little different though, since that's a section, and this `tags` is a list field. I couldn't find anything in the GitHub documentation about how it treats an empty list here. I did find some discussions where someone asked how to run with every tag, and all the solutions say to have the `- "**"` entry.

Reply:
Good find! Added.